src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,26 +142,37 @@
 ;;; Public
 
 (schema/defn ^:always-validate
+  initialize-lifecycle-fns :- jruby-schemas/LifecycleFns
+  [config :- (schema/maybe {schema/Keyword schema/Any})]
```

Comment:
yep, will change... although it does match with what `initialize-config`, below, does.